.github/workflows/tests.yaml

```diff
@@ -17,53 +16,38 @@ jobs:
         - all-build
         - linux-amd64-grpcproxy
         - linux-386-unit
-        exclude:
-        - go: 1.12.17
-          target: linux-amd64-grpcproxy
-        - go: 1.12.17
-          target: linux-386-unit
-        - go: 1.15.15
-          target: linux-amd64-integration-1-cpu
-        - go: 1.15.15
-          target: linux-amd64-integration-2-cpu
-        - go: 1.12.17
-          target: linux-amd64-unit
-        - go: 1.15.15
-          target: linux-amd64-coverage
-        - go: 1.12.17
-          target: linux-amd64-fmt
-        - go: 1.15.15
-          target: linux-386-unit
     steps:
     - uses: actions/checkout@v2
     - uses: actions/setup-go@v2
       with:
-        go-version: ${{ matrix.go }}
+        go-version: "1.16.15"
     - run: date
     - env:
         TARGET: ${{ matrix.target }}
-        GO_VERSION: ${{ matrix.go }}
       run: |
-        RACE='true'; if [[ ${GO_VERSION} == 1.15.15 ]]; then echo 'setting race off'; RACE='false'; fi
+        go version
+        echo ${GOROOT}
+        sudo unlink /usr/bin/go; sudo ln -s ${GOROOT}/bin/go /usr/bin/go; ls -lrt /usr/bin/go
+
         echo "${TARGET}"
         case "${TARGET}" in
           linux-amd64-fmt)
             GOARCH=amd64 PASSES='fmt bom dep' ./test
             ;;
           linux-amd64-integration-1-cpu)
-            GOARCH=amd64 CPU=1 PASSES='integration' RACE="${RACE}" ./test
+            GOARCH=amd64 CPU=1 PASSES='integration' RACE='false' ./test
             ;;
           linux-amd64-integration-2-cpu)
-            GOARCH=amd64 CPU=2 PASSES='integration' RACE="${RACE}" ./test
+            GOARCH=amd64 CPU=2 PASSES='integration' RACE='false' ./test
             ;;
           linux-amd64-integration-4-cpu)
-            GOARCH=amd64 CPU=4 PASSES='integration' RACE="${RACE}" ./test
+            GOARCH=amd64 CPU=4 PASSES='integration' RACE='false' ./test
             ;;
           linux-amd64-functional)
-            ./build && GOARCH=amd64 PASSES='functional' RACE="${RACE}" ./test
+            ./build && GOARCH=amd64 PASSES='functional' ./test
             ;;
           linux-amd64-unit)
-            GOARCH=amd64 PASSES='unit' RACE="${RACE}" ./test
+            GOARCH=amd64 PASSES='unit' RACE='false' ./test
```

Comment:
I think we should have at least one test running with `RACE='true'`

Reply:
Yes, I agree. But let's add any new pipeline in a separate PR. I just bumped the go version to 1.16.15 in this PR, and keep all others unchanged. 

Previously when go version is 1.15.15, then RACE is set to false, see [tests.yaml#L47](https://github.com/etcd-io/etcd/blob/c2c9e7de019661b61f86e1743f461090c1b10771/.github/workflows/tests.yaml#L47). So I just keep it as it's.